var/spack/repos/builtin/packages/spark/package.py

```diff
@@ -0,0 +1,36 @@
+from spack import *
+
+import distutils
+from distutils import dir_util
+
+
+class Spark(Package):
+    """
+    Apache Spark is a fast and general engine
+    for large-scale data processing.
+    """
+
+    homepage = "http://spark.apache.org"
+    url      = "http://spark.apache.org/downloads.html"
+
+    variant('hadoop', default=False,
+            description='Build with Hadoop')
+
+    depends_on('jdk')
```

Comment:
@alfredo-gimenez: default (if you don't specify anything) is `type=('build', 'link')`.  So you should mark this build/run.